alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/MonitorRunner.kt

```diff
@@ -105,7 +111,7 @@ class MonitorRunner(
 ) : JobRunner, CoroutineScope, AbstractLifecycleComponent() {
 
     private val logger = LogManager.getLogger(MonitorRunner::class.java)
-
+    private var httpClient: HttpInputClient
```

Comment:
private lateinit var httpClient: HttpInputClient

Reply:
Fixed

Reply:
Initialized in`init()`, so `lateinit` is not needed.